### ISSUE TITLE
Training Code of the Conditional Zero-DCE

### ISSUE THREAD

**Issue by vecbool** [sczhou/LEDNet#15]
Hello, thanks for your works.

I wanted to train the `Conditional Zero-DCE` on a new dataset but finally failed after several attempts. The structure of `Conditional Zero-DCE` has some inconsistencies with the original model, and I don't know how to make the loss functions work.

Could you please release the training code of `Conditional Zero-DCE`? I'm looking forward to your reply.
